tests/constants/admincli/cmd.go

```diff
@@ -79,4 +79,15 @@ const (
 
 	//RemoveVolumes option refers to removing all volumes from a vmgroup
 	RemoveVolumes = " --remove-volumes"
+
+	// CreateVsanPolicy referring to create vsanPolicy
+	// where --name will be name of the vsanPolicy
+	CreateVsanPolicy = vmdkopsAdmin + "policy create --name="
```

Comment:
I don't use that. I think in the current code, no use for "CreatePolicy". I will remove it.